upup/models/cloudup/resources/addons/kube-dns.addons.k8s.io/k8s-1.12.yaml.template

```diff
@@ -130,7 +136,7 @@ spec:
 
       containers:
       - name: kubedns
-        image: k8s.gcr.io/k8s-dns-kube-dns-{{Arch}}:1.14.13
+        image: k8s.gcr.io/k8s-dns-kube-dns:1.15.13
```

Comment:
bootstrapchannelbuilder.go should be updated to match

https://github.com/kubernetes/kops/blob/77663acec2626a275d7bbd396b52331fac1d4ab3/upup/pkg/fi/cloudup/bootstrapchannelbuilder.go#L229

(but only for the k8s 1.12 manifest, not the 1.6 manifest since you're not bumping the version there)

Reply:
Same as above